test/unit/spec/LoginRouter_spec.js

```diff
@@ -175,6 +174,17 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, Logger, OktaAut
       expect(Logger.warn).toHaveBeenCalledWith('Field not defined in schema', arg1);
     }
 
+    var getAutoPushResponse = function (response, autoPushVal) {
+      var responseCopy = Util.deepCopy(response);
+      var factors = responseCopy['response']['_embedded']['factors'];
```

Comment:
nit: Since it is used in 2 places, move `responseCopy['response']['_embedded']` into a var?